ServiceLayer/src/main/java/org/example/ProductCreationService.java

```diff
@@ -0,0 +1,17 @@
+package org.example;
+
```

Comment:
Create Only One Service called ProductService and it should implement all the functionality related to Product